crispycuts/src/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
```

Comment:
Upon renaming this file, you should rename this css file as well and fix the import.